From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Sun, 24 Dec 2023 03:52:13 +0000
Subject: [PATCH] config: Directly reference config apk build target

Adapted from GrapheneOS Vanadium patch 0126-config-Directly-reference-config-apk-build-target.patch for the Vandium browser port.

---
 vandium/BUILD.gn                | 1 +
 vandium/android_config/BUILD.gn | 5 +++++
 2 files changed, 6 insertions(+)

diff --git a/vandium/BUILD.gn b/vandium/BUILD.gn
index 34418da31ed32..51435ef811038 100644
--- a/vandium/BUILD.gn
+++ b/vandium/BUILD.gn
@@ -6,5 +6,6 @@ assert(is_android)
 # An group included in //BUILD.gn to make vandium build target files reachable by GN.
 group("vandium_build_targets") {
   deps = [
+    "android_config:empty_group",
   ]
 }
